.github/workflows/publish_docker.yml

```
@@ -4,8 +4,8 @@ on:
   workflow_dispatch:
     inputs:
       vm_id:
-        description: 'The ID of the VM (binary dst in Docker image)'
-        default: ''
+        description: "The ID of the VM (binary dst in Docker image)"
+        default: ""
```

nit I would revert since it's a bit out of scope - we can later add a yml linter ci step though, to have it consistent.

```suggestion
        description: 'The ID of the VM (binary dst in Docker image)'
        default: ''
```